### ISSUE TITLE
Error querying for a single entity when Id [Key] is Guid

### ISSUE THREAD

**Issue by Misiu** [tdudek1/AutoAPI#24]
I have an entity that has this as a key:
```c#
[Key]
[Column("activityid")]
public Guid Activityid { get; set; }
```

When getting entity by Id I get this error:

> System.InvalidCastException: Invalid cast from 'System.String' to 'System.Guid'.
>    at System.Convert.DefaultToType(IConvertible value, Type targetType, IFormatProvider provider)
>    at System.String.System.IConvertible.ToType(Type type, IFormatProvider provider)
>    at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
>    at AutoAPI.RESTAPIController.Get(RouteInfo routeInfo)
>    at AutoAPI.AutoAPIMiddleware.InvokeAsync(HttpContext context, IRequestProcessor requestProcessor, IAuthorizationService authorizationService)
>    at Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware.Invoke(HttpContext context)
>    at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
>    at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
>    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
>    at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
> [12:38:31 ERR] Connection id "0HMH90CBVJE1O", Request id "0HMH90CBVJE1O:00000007": An unhandled exception was thrown by the application.
> System.InvalidCastException: Invalid cast from 'System.String' to 'System.Guid'.
>    at System.Convert.DefaultToType(IConvertible value, Type targetType, IFormatProvider provider)
>    at System.String.System.IConvertible.ToType(Type type, IFormatProvider provider)
>    at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
>    at AutoAPI.RESTAPIController.Get(RouteInfo routeInfo)
>    at AutoAPI.AutoAPIMiddleware.InvokeAsync(HttpContext context, IRequestProcessor requestProcessor, IAuthorizationService authorizationService)
>    at Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware.Invoke(HttpContext context)
>    at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
>    at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
>    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
>    at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
>    at Microsoft.AspNetCore.Watch.BrowserRefresh.BrowserRefreshMiddleware.InvokeAsync(HttpContext context)
>    at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)

**Comment by tdudek1** [comment]
Thanks for creating the issue.  I'll look into fixing this.

**Comment by tdudek1** [comment]
This is has been fixed with 3.2
